File: pingYouGou-sellerGoods-interface/src/main/java/com/lemonzuo/sellergoods/service/GoodsService.java
package com.lemonzuo.sellergoods.service;

import com.lemonzuo.model.Goods;

/**
 * @author LemonZuo
 * @create 2019-09-11 15:16
 */
public interface GoodsService {
    /**
     * 新增Goods
     * @param goods
     * @return
     */
    Goods addGoods(Goods goods);
}

File: pingYouGou-shop-web/src/main/resources/static/js/service/goodsService.js
app.service("goodsService", function ($http, pingYouGouServer) {
    let url = `${pingYouGouServer}/shopGoods/goods.do`;

    /**
     * 新增商品
     * @param data
     */
    this.addGoods = function (data) {
        return $http.post(url, data);
    }
})